crypto3/libs/blueprint/test/zkevm_bbf/test_l1_wrapper.hpp

```diff
@@ -96,9 +96,9 @@ std::pair<std::vector<std::vector<std::uint8_t>>, std::vector<boost::property_tr
 
 template <typename BlueprintFieldType>
 bool check_proof(
-    nil::blueprint::circuit<zk::snark::plonk_constraint_system<BlueprintFieldType>> bp,
-    assignment<zk::snark::plonk_constraint_system<BlueprintFieldType>> assignment,
-    zk::snark::plonk_table_description<BlueprintFieldType> desc
+    const nil::blueprint::circuit<zk::snark::plonk_constraint_system<BlueprintFieldType>> &bp,
```

Comment:
My be make sense rename file to `test_circuit_builder`

Reply:
Renamed to circuit_test_fixture